action/protocol/staking/delegate.go

```diff
@@ -247,7 +269,55 @@ func (m DelegateMap) Deserialize(buf []byte) error {
 	}
 
 	for _, v := range pb.Delegates {
-		m[hash.BytesToHash160(v.Owner)] = fromProto(v)
+		m[v.Owner] = fromProto(v)
 	}
 	return nil
 }
+
+func stakingGetDelegate(sr protocol.StateReader, name address.Address) (*Delegate, error) {
```

Comment:
why we need to add a prefix "staking" to the function name?

Reply:
to name staking protocol internal API

already added other APIs like stakingGet/PutBucket() etc.

Reply:
as they are internal, the prefix is redundant.